js_modules/dagit/packages/core/src/pipelines/GraphExplorer.tsx

```diff
@@ -160,34 +161,40 @@ export const GraphExplorer: React.FC<GraphExplorerProps> = (props) => {
     [nameMatch, queryResultOps.all],
   );
 
-  const backgroundColor = parentHandle ? ColorsWIP.White : ColorsWIP.White;
-  const backgroundTranslucent = Color(backgroundColor).fade(0.6).toString();
-
   return (
     <SplitPanelContainer
       identifier="explorer"
       firstInitialPercent={70}
       first={
         <>
-          <PathOverlay style={{background: backgroundTranslucent}}>
-            <Breadcrumbs
-              items={explorerPath.opNames.map((name, idx) => ({
-                text: name,
-                onClick: () =>
-                  onChangeExplorerPath(
-                    {...explorerPath, opNames: explorerPath.opNames.slice(0, idx + 1)},
-                    'push',
-                  ),
-              }))}
-              currentBreadcrumbRenderer={() => (
-                <NodeJumpBar
-                  nodes={queryResultOps.all}
-                  nodeType="op"
-                  selectedNode={selectedHandle && selectedHandle.solid}
-                  onChange={handleClickOp}
-                />
-              )}
-            />
+          <PathOverlay>
+            {explorerPath.opNames.length > 1 && (
+              <Breadcrumbs
+                currentBreadcrumbRenderer={() => <span />}
```

Comment:
Did you intentionally kill this out

Reply:
Ahh yeah, for the "non-nested" graph case I think we're going to show nothing, so this will only appear if you go down into a sub-graph, and even then only the parent breadcrumbs.